### PR TITLE
Add exception for WezTerm and Chrome Remote Desktop

### PR DESCRIPTION
**Pull request by kingamajick**
This adds support for WezTerm and Chrome Remote Desktop. Due to the fact that CRD is a Chrome App, it has a unique bundle id created, and as such has to be identified by the file path.

Note: This will only work if CRD is used as a Chrome App, not within side the Chrome Browser directly.

### CODE REVIEW

**Comment by kingamajick** [comment]
Hey, I'm not sure what the failure is for the formatting, checking locally, it all looks good, is there a way to get more detailed logs?

File: jsonnet/lib/file_paths.libsonnet [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  //-----------------------//
+  // FILE PATH IDENTIFIERS //
+  //-----------------------//
+
+  // file path identifiers for remote desktop applications
+  remoteDesktops: [
+    // Chrome Remote desktop
+    'Chrome Remote Desktop\\.app'
```
Comment (rux616) [inline jsonnet/lib/file_paths.libsonnet:9]:
```suggestion
    'Chrome Remote Desktop\\.app',
```

Adding a comma here should fix the formatting acceptance check.

I'll work on getting some better error message around that in the future.

File: jsonnet/lib/karabiner.libsonnet [hunk 2]
```diff
@@ -86,9 +86,14 @@
   //
   // bundles (array, required)
   //   bundle identifiers of applications
-  condition(type, bundles):: {
+  // file_paths (array, option)
```
Comment (rux616) [inline jsonnet/lib/karabiner.libsonnet:89]:
```suggestion

  // file_paths (array, optional)
```

**Review by rux616 — COMMENTED** [review]
Looks pretty good, thanks for the submission! Just a couple things and I think this will be good to merge.

**Review by rux616 — COMMENTED** [review]
Oh yeah, one thing I forgot to mention is that you don't need to update the files in the `json` directory, as they get auto-rendered on a merge to the master branch.

File: jsonnet/lib/file_paths.libsonnet (hunk 1 above)
Comment (rux616) [reply jsonnet/lib/file_paths.libsonnet:9]: > I'll work on getting some better error messaging around that in the future.

While it's not necessary as the offending line is listed above, I just made a change to the format script, but you'll need to close/reopen this PR to make use of it.

**Comment by kingamajick** [comment]
Sorry for slacking on the pull request, let me know if anything else needs to be added to get this merged?

**Review by rux616 — APPROVED** [review]

**Comment by rux616** [comment]
Thanks for your contribution, @kingamajick!

**Pull request merged**